README.md

```diff
@@ -32,40 +32,24 @@ That's it! You are now inside the dev container and have access to all the devel
 
 # Development
 
-The `cdk.json` file tells the CDK Toolkit how to execute your app.
+All the development tools are provided when developing inside the dev container
+(see above). These tools include Python, AWS CLI, AWS CDK CLI, etc. These tools
+also include a Python virtual environment where all the Python packages needed
+are already installed.
 
-This project is set up like a standard Python project.  The initialization
-process also creates a virtualenv within this project, stored under the `.venv`
-directory.  To create the virtualenv it assumes that there is a `python3`
-(or `python` for Windows) executable in your path with access to the `venv`
-package. If for any reason the automatic creation of the virtualenv fails,
-you can create the virtualenv manually.
+If you decide the develop outside of the dev container, some of the development
+tools can be installed by running:
 
-To manually create a virtualenv on MacOS and Linux:
-
-```
-$ python3 -m venv .venv
+```console
+./tools/setup.sh
 ```
 
-After the init process completes and the virtualenv is created, you can use the following
-step to activate your virtualenv.
+Development requires the activation of the Python virtual environment:
 
 ```
 $ source .venv/bin/activate
 ```
 
-If you are a Windows platform, you would activate the virtualenv like this:
-
-```
-% .venv\Scripts\activate.bat
-```
-
-Once the virtualenv is activated, you can install the required dependencies.
-
-```
-$ pip install -r requirements.txt -r requirements-dev.txt
-```
-
 At this point you can now synthesize the CloudFormation template for this code.
```

Comment:
i think we should keep this for those who don't want to develop within a devcontainer, especially those that don't use VS Code or those that don't use any development IDE at all.

Reply:
The documentation says to run `./tools/setup.sh` if developing outside of a dev container, which includes running `pip install`. There is no need to ask the user to run all the setup commands by themselves (`pip instal...`, pre-commit, install CDK, etc.).

Reply:
hmm, good point.  although setup.sh does not tell users to setup a python virtualenv before installing python libraries, which i think is an important thing to do.

Reply:
setup.sh does create a virtual environment before installing the dependencies. The developer just has to active the environment, which is documented in the README.

Reply:
indeed it is documented, sorry i missed it.  Although it's really not required if working within a docker container.